e2e/ios/ReactNativeTHEOplayer/AppDelegate.m

```diff
@@ -0,0 +1,51 @@
+#import "AppDelegate.h"
+
+#import <React/RCTBridge.h>
+#import <React/RCTBundleURLProvider.h>
+#import <React/RCTRootView.h>
+
+#if !TARGET_OS_TV
+#import <GoogleCast/GoogleCast.h>
+#endif
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
```

Comment:
Deze setup is ok als start, maar zal nog een extra pass nodig hebben om ook de new architecture te ondersteunen. Maar laten we dit nu eerst vooral al opstarten onder deze vorm.